zcash_client_backend/CHANGELOG.md

```diff
@@ -81,6 +84,8 @@ and this library adheres to Rust's notion of
     constraint on its `<AccountId>` parameter has been strengthened to `Copy`.
 - `zcash_client_backend::fees`:
   - Arguments to `ChangeStrategy::compute_balance` have changed.
+- `zcash_client_backend::scanning`:
+  - `testing::fake_compact_block` is now public.
```

Comment:
Duplicated from the "Added" section (the fact that it was crate-internal previously is irrelevant to consumers of the public API; this is an addition).
```suggestion
```